membership-attribute-service/app/models/Attributes.scala

```diff
@@ -53,15 +54,16 @@ case class Attributes(
   lazy val contentAccess = ContentAccess(
     member = isPaidTier || isFriendTier,
     paidMember = isPaidTier,
-    recurringContributor = isContributor,
+    recurringContributor = isRecurringContributor,
     digitalPack = digitalSubscriberHasActivePlan,
     paperSubscriber = isPaperSubscriber,
     guardianWeeklySubscriber = isGuardianWeeklySubscriber
   )
 
   // show support messaging (in app & on dotcom) if they do NOT have any active products
   // TODO in future this could become more sophisticated (e.g. two weeks before their products expire)
-  lazy val showSupportMessaging = !(isPaidTier || isContributor || digitalSubscriberHasActivePlan || isPaperSubscriber || isGuardianWeeklySubscriber)
+  lazy val showSupportMessaging =
+    !(isPaidTier || isRecurringContributor || isOneOffContributor || digitalSubscriberHasActivePlan || isPaperSubscriber || isGuardianWeeklySubscriber)
```

Comment:
The current rule is to not show asks to people who have made a contribution *in the past 6 months*, so maybe `isRecentOneOffContributor` would be more appropriate.